builder/spec/cpp/cpp_builder_spec.rb

```diff
@@ -1,4 +1,5 @@
 require_relative '../../cpp_builder'
+require_relative '../../../aggregate/boost_test_parser'
```

Comment:
Nit: I believe this one is not really necessary?

Reply:
Well, I added some tests to check that the detection of aborted tests works:

https://github.com/kaitai-io/kaitai_struct_tests/blob/25b2182b24ba7e1bbc5514e9a676120897930ee0/builder/spec/cpp/cpp_builder_spec.rb#L438-L447

I realize that it's not very clean to add a test of [`aggregate/boost_test_parser.rb`](https://github.com/kaitai-io/kaitai_struct_tests/blob/ac7dd7f068a5ea8e3fb86b2ec9666ff12aeb5b76/aggregate/boost_test_parser.rb) to [`builder/spec/cpp/cpp_builder_spec.rb`](https://github.com/kaitai-io/kaitai_struct_tests/blob/ac7dd7f068a5ea8e3fb86b2ec9666ff12aeb5b76/builder/spec/cpp/cpp_builder_spec.rb), but I was a bit lazy to create a new `*_spec.rb` file under the `aggregate/` folder, sorry :) Feel free to move these `BoostTestParser` tests to the right place.

Reply:
Got it, makes sense then!